.github/CODEOWNERS

```diff
@@ -1,16 +1,16 @@
 bootstrap.sh @deepthi
-/config/mycnf/ @askdba @shlomi-noach
+/config/mycnf/ @deepthi @shlomi-noach @mattlord
 /docker/ @derekperkins @dkhenry @GuptaManan100 @frouioui
 /examples/compose @shlomi-noach
 /examples/demo @sougou
 /examples/legacy_local @deepthi
 /examples/local @rohit-nayak-ps
-/examples/operator @askdba
+/examples/operator @GuptaManan100 @frouioui @mattlord
 /examples/region_sharding @deepthi
 /java/ @harshit-gangal @pH14
```

Comment:
There is a bug here. GitHub complains at this line for some reason. 
```
Unknown owner on line 10: make sure @pH14 exists and has write access to the repository
/java/ @harshit-gangal @pH14
```

Reply:
Looks like he was also a previous maintainer that we should remove from CODEOWNERS.